### PR TITLE
Python SDK Modules

### PR DESCRIPTION
**Pull request by whoarethebritons**
Except for d5fb5cd all changes are from the appengine python 1.8.2 sdk

### CODE REVIEW

File: AppServer/google/appengine/ext/datastore_admin/static/css/compiled.css [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
...t .g-tpl-nest .g-unit,.g-unit .g-unit .g-unit .g-tpl-nest .g-unit{float:left;width:auto;margin:0}.g-tpl-nest-alt .g-unit,.g-unit .g-tpl-nest-alt .g-unit,.g-unit .g-unit .g-tpl-nest-alt .g-unit,.g-unit .g-unit .g-unit .g-tpl-nest-alt .g-unit{float:right;width:auto;margin:0}.goog-button{border-width:1px;border-style:solid;border-color:#bbb #999 #999 #bbb;border-radius:2px;-webkit-border-radius:2px;-moz-border-radius:2px;font:normal normal normal 13px/13px Arial,sans-serif;color:#000;text-align:middle;text-decoration:none;text-shadow:0 1px 1px rgba(255,255,255,1);background:#eee;background-image:-webkit-gradient(linear,0 0,0 100%,from(#fff),to(#ddd));background:-moz-linear-gradient(top,#fff,#ddd);filter:progid:DXImageTransform.Microsoft.Gradient(EndColorstr='#dddddd',StartColorstr='#ffffff',GradientType=0);cursor:pointer;margin:0;display:inline;display:-moz-inline-box;display:inline-block;*overflow:visible;padding:4px 8px 5px}a.goog-button,span.goog-button,div.goog-button{padding:4px 8px 5px}.goog-button:visited{color:#000}.goog-button{*display:inline}.goog-button:focus,.goog-button:hover{border-color:#000}.goog-button:active,.goog-button-active{color:#000;background-color:#bbb;border
```
Comment (cdonati) [inline AppServer/google/appengine/ext/datastore_admin/static/css/compiled.css]: Please limit lines to 80 characters. (jk)

File: AppServer/google/appengine/tools/devappserver2/devappserver2.py [hunk 2]
```diff
@@ -500,7 +500,7 @@ def _clear_search_indexes_storage(search_index_path):
                       e)
 
 
-def _setup_environ(app_id):
+def _setup_environ(app_id, version_id): # AS: set
```
Comment (cdonati) [inline AppServer/google/appengine/tools/devappserver2/devappserver2.py]: cryptic comment

File: AppServer/google/appengine/tools/devappserver2/module.py [hunk 3]
```diff
@@ -35,17 +34,17 @@
 import urlparse
 import wsgiref.headers
 
-
 from google.appengine.api import api_base_pb
 from google.appengine.api import apiproxy_stub_map
 from google.appengine.api import appinfo
 from google.appengine.api import request_info
-from google.appengine.api.logservice import logging_capnp
```
Comment (cdonati) [inline AppServer/google/appengine/tools/devappserver2/module.py]: Is this (and capnp) no longer needed for the log service?

File: AppServer/google/appengine/api/images/images_stub.py [hunk 4]
```diff
@@ -145,8 +145,8 @@ def __init__(self, service_name="images", host_prefix=""):
 
     Args:
       service_name: Service name expected for all calls.
-      host_prefix: the URL prefix (protocol://host) to preprend to image urls
-        on a call to GetUrlBase.
+      host_prefix: the URL prefix (protocol://host:port) to prepend to
+        image urls on a call to GetUrlBase.
```
Comment (whoarethebritons) [inline AppServer/google/appengine/api/images/images_stub.py]: I just accepted their comment, I can change it back

File: AppServer/google/appengine/tools/devappserver2/devappserver2.py (hunk 2 above)
Comment (whoarethebritons) [inline AppServer/google/appengine/tools/devappserver2/devappserver2.py]: haha whoops

File: AppServer/google/appengine/tools/devappserver2/module.py (hunk 3 above)
Comment (whoarethebritons) [inline AppServer/google/appengine/tools/devappserver2/module.py]: As discussed, I had run our hawkeye logservice test on this branch and it passed. capnp is still a requirement, but this file seems to only need the stub. I will look into it a bit more though.

**Comment by whoarethebritons** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3568/

File: AppServer/google/appengine/tools/devappserver2/application_configuration.py [hunk 5]
```diff
@@ -114,22 +114,22 @@ def api_version(self):
...
   @property
-  def server_name(self):
-    return self._server_name or 'default'
+  def module_name(self):
+    return self._module_name or 'default'
 
   @property
   def major_version(self):
-    return self._version
+    return self._version or 'default' # AS: set default if not set.
```
Comment (jeanleonov) [inline AppServer/google/appengine/tools/devappserver2/application_configuration.py]: [PEP8](https://www.python.org/dev/peps/pep-0008/#inline-comments) recommends to put at least 2 spaces before inline-comment.

**Pull request merged**